archinstall/lib/user_interaction/save_conf.py

```diff
@@ -1,9 +1,13 @@
 from __future__ import annotations
 
+import archinstall
```

Comment:
Avoid importing like this, as it's not a relative import - it might import from outside sources.

Reply:
Yea, I know vscode and other IDE's love to add `import archinstall` because it auto detects that this is where `log` is exposed hehe.